.github/workflows/test.yml

```diff
@@ -119,9 +119,29 @@ jobs:
       - name: Unit Tests
         run: yarn test:ci
 
+  cli-install-test:
+    runs-on: ubuntu-latest
+    needs:
+      - yarn-install
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
+      - name: install-hyperlane-cli
+        id: install-hyperlane-cli
+        uses: ./.github/actions/install-cli
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
+      - name: Test run the CLI
+        run: hyperlane --version
```

Comment:
i don't think test is required now that cli-e2e uses the packed/installed cli

Reply:
Wouldn't it be better to leave it for clarity so that instead of having to go to the CI logs, we can just know why tests are failing by knowing that the single cli install action failed?

Reply:
in that case, can we make cli-e2e not depend on this test? otherwise that's an extra 3m added to each CI run for no good reason

Reply:
so cli-e2e-matrix and cli-install-test can both run in parallel and overall ci time is unchanged

Reply:
sounds good!